packages/terra-application-navigation/src/common/PopupMenu.module.scss

```
@@ -0,0 +1,22 @@
+:local {    
```

nit: remove extra whitespace
```suggestion
:local {
```

---

Updated.